module/module_test.go

```diff
@@ -484,6 +484,8 @@ func TestAttributeConversion(t *testing.T) {
 	}
 
 	t.Run("non-reconfigurable creation", func(t *testing.T) {
+		ctx := context.Background()
```

Comment:
`module/example_test.go` [defined](https://github.com/viamrobotics/rdk/blob/main/module/example_test.go#L21) a globally scoped `ctx` var set to `context.Background()` that a few other tests in the `module` package were relying on. I just added `ctx := context.Background()` to all of those tests.